lms/djangoapps/branding/views.py

```diff
@@ -313,13 +312,24 @@ def footer(request):
     else:
         return HttpResponse(status=406)
 
+
 @require_http_methods(['GET', 'POST'])
+@check_recaptcha
 def contact_form(request):
+    """
+    Render 'contact' page.
+    """
     sent_mail = False
     form = ContactForm()
+    is_use_google_recaptcha = configuration_helpers.get_value('USE_GOOGLE_RECAPTCHA', settings.USE_GOOGLE_RECAPTCHA)
+    # This parameter contains 'True' as the default value.
```

Comment:
please add keyword NOTE(andrey.lykhoman) and add more than one space indent in the relational comment's strings for better highlighting in the IDE

Reply:
I added NOTE highlight.